packages/connect-thegraph-template/subgraph/abis/DAOKit.json

```diff
@@ -0,0 +1,293 @@
+[
```

Comment:
ABIs: I'm just more comfortable with using whatever ABI etherscan shows when I go to the address of a given data source. Facu recommended to fetch or generate the ABIs ourselves, but it would be really complicated here because we would have to add a lot of dependencies.

Build: Build files should not be uploaded to git. If they are, it's an error.